examples/resources/kentik-synthetic_test-bulk-create/agents_filtered_by_asn/resource.tf

```diff
@@ -23,31 +23,23 @@ resource "kentik-synthetics_test" "agents-filtered-by-asn-test" {
       "traceroute"
     ]
     monitoring_settings {
-      activation_grace_period = "2"
       activation_time_unit    = "m"
       activation_time_window  = "5"
       activation_times        = "3"
     }
     ping {
       period = 60
-      count  = 5
-      expiry = 3000
     }
     trace {
       period   = 60
-      count    = 3
       protocol = "udp"
-      port     = 33434
-      expiry   = 22500
-      limit    = 30
     }
-    port     = 443
-    protocol = "icmp"
+    protocol = "tcp"
```

Comment:
TCP ping (which is what `settings.protocol = tcp` enables), requires `port`. Unless the TF provider or the SDK default it to some other value, it is set to 0, which is not really a usable value. API validation should probably refuse the create request, but currently does not :-(.

Reply:
Added port 443 to all examples, so that they are more realistic. Left port to be optional, tough